app/test_main.py

```diff
@@ -1,3 +1,16 @@
+import pytest
 from app.main import get_human_age
 
-# write your code here
+
+@pytest.mark.parametrize("cat_age, dog_age, expected", [
+    (0, 0, [0, 0]),
+    (14, 14, [0, 0]),
+    (15, 15, [1, 1]),
+    (23, 23, [1, 1]),
+    (24, 24, [2, 2]),
+    (27, 27, [2, 2]),
+    (28, 28, [3, 2]),
+    (100, 100, [21, 17]),
```

Comment:
Add a test case for negative values and a test case with different values for `cat_age` and `dog_age` within one test to cover the requirements of the [checklist](https://github.com/Artem-user-19/py-cat-and-dog-years/blob/develop/checklist.md#clean-code).